File: URI/1 - INICIANTE/C/1066 - ParesImparesPositivosNegativos.c
#include <stdio.h>

int main()
{
	int x = 0, par = 0, impar = 0, negativo = 0, positivo = 0, num;
	
	while (x < 5)
	{
		scanf(" %d", &num);
		if (num > 0){positivo += 1;}
		if (num < 0){negativo += 1;}
		if (num % 2 == 0){par += 1;}
		if (num % 2 != 0){impar += 1;}
		x ++;
	}
	
	printf("%d valor(es) par(es)\n", par);
	printf("%d valor(es) impar(es)\n", impar);
	printf("%d valor(es) positivo(s)\n", positivo);
	printf("%d valor(es) negativo(s)\n", negativo);
	
	return 0;
}

File: URI/1 - INICIANTE/Python/1066 - ParesImparesPositivosNegativos.py

pares = impares = positivos = negativos = 0
x = 1
while x <= 5:
    a = int(input())
    if a > 0:
        positivos += 1
    if a < 0:
        negativos += 1
    if a % 2 == 0:
        pares += 1
    if a % 2 != 0:
        impares += 1

    x+=1

print('{} valor(es) par(es)'.format(pares))
print('{} valor(es) impar(es)'. format(impares))
print('{} valor(es) positivo(s)'.format(positivos))
print('{} valor(es) negativo(s)'.format(negativos))